scripts/build/setup/termux_setup_nim.sh

```diff
@@ -0,0 +1,59 @@
+termux_setup_nim() {
+
+	if [[ -z "${TERMUX_NIM_VERSION-}" ]]; then
+		TERMUX_NIM_VERSION=$(
+			. "${TERMUX_SCRIPTDIR}/packages/nim/build.sh"
+			echo ${TERMUX_PKG_VERSION}
+		)
+	fi
+
+	if [[ "${TERMUX_ON_DEVICE_BUILD}" == "true" ]]; then
+		if [[ -z "$(command -v nim)" ]]; then
```

Comment:
this is inconsistent with how we check if nim is available from how it is done in `termux_setup_nodejs` or any other setup scripts.